File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/DomainOrderResponseAttributes.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

class DomainOrderResponseAttributes
{
    /**
     * Registrar order ID.
     */
    #[DTA\Data(field: "id", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $id = null;

    /**
     * Administrative contact email provided for the order.
     */
    #[DTA\Data(field: "admin_email", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $admin_email = null;

}

File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/DomainOrderResponseAttributes.scala

package org.openapitools.client.model


case class DomainOrderResponseAttributes (
    /* Registrar order ID. */
    _id: Option[String],
    /* Administrative contact email provided for the order. */
    _admin_email: Option[String]
)
object DomainOrderResponseAttributes {
    def toStringBody(var_id: Object, var_admin_email: Object) =
        s"""
        | {
        | "id":$var_id,"admin_email":$var_admin_email
        | }
        """.stripMargin
}
